tests/src/rules/no-self-import.js

```diff
@@ -0,0 +1,42 @@
+import { test, testFilePath } from '../utils'
+
+import { RuleTester } from 'eslint'
+
+const ruleTester = new RuleTester()
+    , rule = require('rules/no-self-import')
+
+const error = {
+  ruleId: 'no-self-import',
+  message: 'Module imports itself',
+}
+
+ruleTester.run('no-self-import', rule, {
+  valid: [
+    test({ code: 'import _ from "lodash"'}),
```

Comment:
Thanks, that makes sense, I don't totally grok the testing setup yet. I'll add that.